thrift/admin.thrift

```diff
@@ -217,3 +247,40 @@ struct DescribeClusterResponse {
   10: optional shared.SupportedClientVersions supportedClientVersions
   20: optional MembershipInfo membershipInfo
 }
+
+enum QueueType {
+  Replication,
+  Domain,
+}
+
+struct ReadDLQMessagesRequest{
+  10: optional QueueType queueType
+  20: optional i32 shardID
```

Comment:
If the shardID doesn't exist, shall we return `EntityNotExistError`?

Reply:
Correct. Added the EntityNotExistError